woocommerce/woocommerce-ios
# Lightweight Storefront: Update profiler flow

Closes: #11407 


## Description
Removes the following challenges and features questions from the store creation profiler flow.

| Challenges | Features |
|--------|--------|
| ![Simulator Screenshot - iPhone 14 Pro - 2023-12-11 at 17 26 40](https://github.com/woocommerce/woocommerce-ios/assets/524475/82a342e9-e2ec-44bb-9df4-80222ba24dbe) | ![Simulator Screenshot - iPhone 14 Pro - 2023-12-11 at 17 26 43](https://github.com/woocommerce/woocommerce-ios/assets/524475/ed146393-cdba-4937-8f79-5e5628dba5ef) | 

I am not checking for `lightweightStorefront` feature flag as removing these profile questions change is not related to theme selection. 

## Testing instructions


- Install the app and launch it
- Logout
- Tap on the "Create a New Store" button
- Login using your WPCOM account
- Tap "Try For Free" to start store creation flow
- From the profiler flow select answers
- After selecting a country, you should be redirected to the loading screen.
- Check Xcode console and validate that the following event is tracked with your selected answers.
```
Tracked site_creation_profiler_data, properties: [AnyHashable("user_commerce_journey"): "im_already_selling", AnyHashable("industry_slug"): "food_and_drink", AnyHashable("country_code"): "AD"]
```

## Screenshots

New profiler flow with challenges and features questions removed

https://github.com/woocommerce/woocommerce-ios/assets/524475/123b3b38-29e1-4ab0-9e99-15ea540da3d4



---
